protostar-rust/src/running.rs

```diff
@@ -27,7 +27,7 @@ fn build_hints_dict<'b>(
         if !instruction.hints.is_empty() {
             // Register hint with string for the hint processor.
             for hint in &instruction.hints {
-                string_to_hint.insert(hint.to_string(), hint.clone());
+                string_to_hint.insert(format!("{hint:?}"), hint.clone());
```

Comment:
Why we choose this way instead of dumping to string?

Reply:
Display traits supposedly no longer exists for hint, so it cannot be used.

Reply:
`Display` trait supposedly is no longer available. Created #2180 to investigate an alternative solution.